core/server/master/src/main/java/alluxio/master/metastore/caching/CachingInodeStore.java

```diff
@@ -299,6 +299,23 @@ public InodeCache(CacheConfiguration conf) {
           MetricKey.MASTER_INODE_CACHE_MISSES, MetricKey.MASTER_INODE_CACHE_SIZE);
     }
 
+    @Override
+    protected void onPut(
```

Comment:
is it possible to avoid modifying the Cache interface? I prefer to keep this a pure logging change and keep the risk very low, and for that I'm willing to give up some observability.

Reply:
the interface change is quite trivial. we just change the interface to pass more params, rather than doing any real behavior change.